### PR TITLE
Update Install.md

### PR DESCRIPTION
**Pull request by copiis**
Aded hint for Apache instal

### CODE REVIEW

**Comment by nupplaphil** [comment]
Can you plz rebase your change to the `2019.12-rc` branch so the upcoming release will contain it?

File: doc/Install.md [hunk 1]
```diff
@@ -112,8 +112,8 @@ Please check the [troubleshooting](#Troubleshooting) section if running on MySQL
 
 ### Option A: Run the installer
 
-Point your web browser to the new site and follow the instructions.
-Please note any error messages and correct these before continuing.
+Before you point your web browser to the new site you need to rename .htaccess-dist to .htaccess for Apache installs.
```
Comment (tobiasd) [inline doc/Install.md]: Could you please encase the file names `.htaccess-dist` and `.htaccess` in `` so that they are marked as inline "code"?

**Comment by tobiasd** [comment]
I changed the targeted branch to the `2019.12-rc` branch; @copiis you should not need to do anything about the branch.

File: doc/Install.md [hunk 2]
```diff
@@ -112,8 +112,8 @@ Please check the [troubleshooting](#Troubleshooting) section if running on MySQL
...
+Follow the instructions. Please note any error messages and correct these before continuing.
```
Comment (nupplaphil) [inline doc/Install.md]: Plz add a Line-Break after the dot (`.`). Every sentence should be a new line for better reviewing

**Pull request merged**